test/integration/services/alert.js

```diff
@@ -0,0 +1,89 @@
+describe('Alert Service', function() {
+  specify('Displaying', function() {
+    cy
+      .visit('/');
+
+    cy
+      .clock();
+
+    cy
+      .window()
+      .should('have.property', 'Radio')
+      .then(Radio => {
+        Radio.request('alert', 'show:success', 'success');
+      });
+
+    cy
+      .get('.alert-box')
+      .contains('success')
+      .should('exist')
+      .tick(4900);
+
+    cy
+      .get('.alert-box')
+      .should('not.exist');
+
+    cy
+      .clock()
+      .then((clock) => {
+        clock.restore();
+      });
+
+    cy
+      .window()
+      .should('have.property', 'Radio')
+      .then(Radio => {
+        Radio.request('alert', 'show:info', 'info');
+      });
+
+    cy
+      .get('.alert-box')
+      .contains('info');
+
+    cy
```

Comment:
now that we're testing components a lot more, we're using this pattern everywhere.  I think it's time we make a custom command.  I'm cool with doing it in this PR or we can do it in a next PR.. up to you, but ideally the API would be:
```js
cy
  .getRadio(Radio => {
    Radio.request(...
  });
```